payments-core/src/main/java/com/stripe/android/payments/core/authentication/IntentAuthenticatorRegistry.kt

```diff
@@ -0,0 +1,16 @@
+package com.stripe.android.payments.core.authentication
+
+import com.stripe.android.model.StripeIntent
+
+/**
+ * Registry to map [StripeIntent] to the corresponding [IntentAuthenticator] to handle its next_action
+ */
+internal interface IntentAuthenticatorRegistry {
```

Comment:
You could make this a `fun interface`

Reply:
we'll add a new #register method for new `Intent`, `Authenticator` pairs in a follow up change